resources/assets/js/app.js

```diff
@@ -29,6 +29,7 @@ require('./load-remote.js');
 require('cropperjs');
 
 $(document).ready(function(){
+    $("#dtBox").DateTimePicker();
```

Comment:
Probably doesn't matter, but why change the position of the initialization call? Also, I'm not familiar with the package you're suggesting.  I'm guessing it uses a single element to hold the modal/popup instead of the individual fields, which is why you added the <div> below and are using `#dtBox` instead of `.datePicker`

Reply:
Correct, it uses a single element to hold the element.

It looks like this: 
![screenshot from 2017-11-09 02 29 45](https://user-images.githubusercontent.com/2234034/32582132-ea9a4f46-c4f5-11e7-8885-e3bc4965073e.png)


The new location is an oversight, as I just needed to make sure it was in document.ready for my tests, and forgot to move it once I had concluded my tests.